server/controllers/CommentController.js

```diff
@@ -28,7 +33,15 @@ class CommentController {
       if (!article) {
         return response(res, 404, 'failure', 'Article not found', null, null);
       }
-      await Comment.create({ content, userId, articleId });
+      if (!content) {
+        return response(res, 400, 'failure', 'Enter a comment', null, null);
+      }
+      await Comment.create({
+        id,
+        content,
+        userId,
+        articleId
+      });
       const getComment = await Comment.findOne({
         where: {
           articleId,
```

Comment:
Nice work TJ! 
I feel you should not need to getComment from the DB in order to confirm comment creation. You could just do
`const commentCreate = await Comment.create({
        id,
        content,
        userId,
        articleId
      });`
and check 
`if(commentCreate)`